lib/features/home/data/repository/introduced_profile_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:atwoz_app/core/network/base_repository.dart';
+import 'package:atwoz_app/core/util/util.dart';
+import 'package:atwoz_app/features/home/data/dto/introduced_profile_dto.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final introducedProfileRepositoryProvider =
+    Provider<IntroducedProfileRepository>((ref) {
+  return IntroducedProfileRepository(ref);
+});
+
+class IntroducedProfileRepository extends BaseRepository {
+  IntroducedProfileRepository(Ref ref) : super(ref, '/member/introduction');
+
+  Future<List<IntroducedProfileDto>> getProfiles(String category) async {
+    final res =
+        await apiService.getJson('$path/$category', requiresAuthToken: true);
+
+    final data = res['data'];
+    if (data == null || data is! List) {
```

Comment:
```suggestion
    if (data is! List<Map<String, dynamic>>) {
```
위 타입 체크가 더 안정적일 것 같은데 검토 부탁드려요